### PR TITLE
Further GQL documentation

### PR DESCRIPTION
**Pull request by JPryce-Aklundh**

### CODE REVIEW

**Comment by neo-technology-commit-status-publisher** [comment]
This PR includes documentation updates
View the updated docs at https://neo4j-docs-cypher-965.surge.sh

Updated pages:
- [CREATE](https://neo4j-docs-cypher-965.surge.sh/cypher-manual/5/clauses/create)
- [DELETE](https://neo4j-docs-cypher-965.surge.sh/cypher-manual/5/clauses/delete)
- [FINISH](https://neo4j-docs-cypher-965.surge.sh/cypher-manual/5/clauses/finish)
- [UNION](https://neo4j-docs-cypher-965.surge.sh/cypher-manual/5/clauses/union)
- [Aggregating functions](https://neo4j-docs-cypher-965.surge.sh/cypher-manual/5/functions/aggregating)
- [Scalar functions](https://neo4j-docs-cypher-965.surge.sh/cypher-manual/5/functions/scalar)
- [Concepts](https://neo4j-docs-cypher-965.surge.sh/cypher-manual/5/patterns/concepts)
- [Syntax and semantics](https://neo4j-docs-cypher-965.surge.sh/cypher-manual/5/patterns/reference)
- [Operators](https://neo4j-docs-cypher-965.surge.sh/cypher-manual/5/syntax/operators)

**Comment by neo-technology-commit-status-publisher** [comment]
This PR includes documentation updates
View the updated docs at https://neo4j-docs-cypher-965.surge.sh

Updated pages:
- [CREATE](https://neo4j-docs-cypher-965.surge.sh/cypher-manual/5/clauses/create)
- [DELETE](https://neo4j-docs-cypher-965.surge.sh/cypher-manual/5/clauses/delete)
- [FINISH](https://neo4j-docs-cypher-965.surge.sh/cypher-manual/5/clauses/finish)
- [UNION](https://neo4j-docs-cypher-965.surge.sh/cypher-manual/5/clauses/union)
- [Aggregating functions](https://neo4j-docs-cypher-965.surge.sh/cypher-manual/5/functions/aggregating)
- [Scalar functions](https://neo4j-docs-cypher-965.surge.sh/cypher-manual/5/functions/scalar)
- [Concepts](https://neo4j-docs-cypher-965.surge.sh/cypher-manual/5/patterns/concepts)
- [Syntax and semantics](https://neo4j-docs-cypher-965.surge.sh/cypher-manual/5/patterns/reference)
- [Operators](https://neo4j-docs-cypher-965.surge.sh/cypher-manual/5/syntax/operators)

File: modules/ROOT/pages/clauses/union.adoc [hunk 1]
```diff
@@ -98,6 +98,7 @@ The combined result is returned, without duplicates.
 === UNION DISTINCT
 
 Removal of duplicates can also be accomplished by explicitly including `DISTINCT` in the `UNION`.
+The `UNION DISTINCT` keyword was introduced as part of Cypher's xref:appendix/gql-conformance/index.adoc[], and using it is functionally the same as using simple `DISTINCT`.
```
Comment (gem-neo4j) [inline modules/ROOT/pages/clauses/union.adoc]:
```suggestion
The `UNION DISTINCT` keyword was introduced as part of Cypher's xref:appendix/gql-conformance/index.adoc[], and using it is functionally the same as using simple `UNION`.
```

File: modules/ROOT/pages/functions/aggregating.adoc [hunk 2]
```diff
@@ -376,6 +376,7 @@ To avoid counting duplicates, use the `DISTINCT` keyword.
 
 As of Neo4j 5.15, it is also possible to use the `ALL` keyword with aggregating functions.
 This will count all results, including duplicates, and is functionally the same as not using the `DISTINCT` keyword.
+The `ALL`keyword was introduced as part of Cypher's xref:appendix/gql-conformance/index.adoc[].
```
Comment (gem-neo4j) [inline modules/ROOT/pages/functions/aggregating.adoc]:
```suggestion
The `ALL` keyword was introduced as part of Cypher's xref:appendix/gql-conformance/index.adoc[].
```

**Review by gem-neo4j — APPROVED** [review]
Great! Just some minor comments :)

**Comment by neo-technology-commit-status-publisher** [comment]
This PR includes documentation updates
View the updated docs at https://neo4j-docs-cypher-965.surge.sh

Updated pages:
- [CREATE](https://neo4j-docs-cypher-965.surge.sh/cypher-manual/5/clauses/create)
- [DELETE](https://neo4j-docs-cypher-965.surge.sh/cypher-manual/5/clauses/delete)
- [FINISH](https://neo4j-docs-cypher-965.surge.sh/cypher-manual/5/clauses/finish)
- [UNION](https://neo4j-docs-cypher-965.surge.sh/cypher-manual/5/clauses/union)
- [Aggregating functions](https://neo4j-docs-cypher-965.surge.sh/cypher-manual/5/functions/aggregating)
- [Scalar functions](https://neo4j-docs-cypher-965.surge.sh/cypher-manual/5/functions/scalar)
- [Concepts](https://neo4j-docs-cypher-965.surge.sh/cypher-manual/5/patterns/concepts)
- [Syntax and semantics](https://neo4j-docs-cypher-965.surge.sh/cypher-manual/5/patterns/reference)
- [Operators](https://neo4j-docs-cypher-965.surge.sh/cypher-manual/5/syntax/operators)

**Comment by neo-technology-commit-status-publisher** [comment]
Thanks for the documentation updates.

The preview documentation has now been torn down - reopening this PR will republish it.

**Pull request merged**